src/io/flutter/module/settings/FlutterCreateAddtionalSettingsFields.java

```diff
@@ -0,0 +1,81 @@
+
+    descriptionField = new JTextField();
+    descriptionField.setToolTipText(FlutterBundle.message("flutter.module.create.settings.description.tip"));
+    descriptionField.setText(FlutterBundle.message("flutter.module.create.settings.description.default_text"));
+
+    androidLanguageRadios = new RadiosForm(FlutterBundle.message("flutter.module.create.settings.radios.android.java"),
+                                           FlutterBundle.message("flutter.module.create.settings.radios.android.kotlin"));
+    androidLanguageRadios.setToolTipText(FlutterBundle.message("flutter.module.create.settings.radios.android.tip"));
+
+    iosLanguageRadios = new RadiosForm(FlutterBundle.message("flutter.module.create.settings.radios.ios.object_c"),
+                                       FlutterBundle.message("flutter.module.create.settings.radios.ios.swift"));
+    iosLanguageRadios.setToolTipText(FlutterBundle.message("flutter.module.create.settings.radios.ios.tip"));
+
+    includeDriverTextField = new JCheckBox();
+    includeDriverTextField.setSelected(true);
+    includeDriverTextField.setToolTipText(FlutterBundle.message("flutter.module.create.settings.includedriver.tip"));
+  }
+
+  public void addSettingsFields(@NotNull SettingsStep settingsStep) {
+    settingsStep.addSettingsField("", new JLabel(" "));
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.radios.type.label"),
+                                  projectTypeRadios.getComponent());
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.radios.org.label"), orgField);
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.description.label"), descriptionField);
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.radios.android.label"),
+                                  androidLanguageRadios.getComponent());
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.radios.ios.label"),
+                                  iosLanguageRadios.getComponent());
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.includedriver.label"), includeDriverTextField);
+
+    settingsStep.addSettingsField("", new JLabel(" "));
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.help.label"), new JLabel(" "));
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.help.project_name.label"),
+                                  new JLabel(FlutterBundle.message("flutter.module.create.settings.help.project_name.description")));
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.help.org.label"),
+                                  new JLabel(FlutterBundle.message("flutter.module.create.settings.help.org.description")));
```

Comment:
I love this.  I wonder if we could do something to set this help apart a bit more. Maybe putting it in it's own group (sorry, that's an eclipse/SWT-ism... not sure what that's called in Swing).  Or having a dividing line or styled text?  Maybe we could add a hyperlink to get more info as well?  (Oops: I guess that assumes there's a good page with `flutter create` docs but I don't see one.  cc @LarkAscending in case I'm missing one.